apps/files/lib/Service/OwnershipTransferService.php

```diff
@@ -194,7 +194,7 @@ protected function analyse(string $sourceUid,
 		$output->writeln('Validating quota');
 		$size = $view->getFileInfo($sourcePath, false)->getSize(false);
 		$freeSpace = $view->free_space($destinationUid . '/files/');
-		if ($size > $freeSpace) {
+		if ($size > $freeSpace && $freeSpace !== -2) {
```

Comment:
```suggestion
		if ($size > $freeSpace && $freeSpace !== FileInfo::SPACE_UNKNOWN) {
```

To make it more readable?

Reply:
Yes please!